apps/extension/src/background/vault/tests/service.test.ts

```diff
@@ -45,7 +46,12 @@ describe("Testing untouched Vault Service", () => {
     const sessionStore = new KVStoreMock<SessionPassword>(
       KVPrefix.SessionStorage
     );
-    service = new VaultService(storage, sessionStore, cryptoMemory);
+
+    const localStorage = new LocalStorage(
+      new KVStoreMock(KVPrefix.LocalStorage)
+    );
+    const sdkService = await SdkService.init(localStorage);
+    service = new VaultService(storage, sessionStore, sdkService, cryptoMemory);
```

Comment:
It looks like cryptoMemory is unused in VaultService so we can remove it from constructor :)

Reply:
Thanks, got it!